OpenSearch: 'monitor_query_terms' is not defined

The following error needs to be resolved:

```bash
[ERROR] NameError: name 'monitor_query_terms' is not defined
Traceback (most recent call last):
  File "/var/lang/lib/python3.8/imp.py", line 234, in load_module
    return load_source(name, filename, file)
  File "/var/lang/lib/python3.8/imp.py", line 171, in load_source
    module = _load(spec)
  File "<frozen importlib._bootstrap>", line 702, in _load
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/var/task/lambda.py", line 16, in <module>
    from set_configuration import (
  File "/var/task/set_configuration.py", line 331, in <module>
    monitor_query_terms,
```

--- a/lambda.py
+++ b/lambda.py
@@ -115,41 +115,6 @@ def remap_index(
     return False
 
 
-def create_alarm(
-    endpoint,
-    awsauth,
-    monitor_name,
-    sns_alert_name,
-    indices,
-    trigger_action_message,
-    trigger_action_subject
-):
-    '''
-
-    create index monitoring alarm using provided 'destination_id', which
-    corresponds to an associated SNS topic
-
-    '''
-
-    monitor_id = ''
-    monitor = get_monitor(endpoint, awsauth, monitor_name)
-    destination_id = get_alert_destination(endpoint, awsauth, sns_alert_name)
-
-    if 'hits' in monitor and 'hits' in monitor['hits']:
-        monitor_id = monitor['hits']['hits'][0]['_index']
-
-    return set_monitor(
-        endpoint,
-        awsauth,
-        monitor_name,
-        destination_id=destination_id,
-        monitor_id=monitor_id,
-        indices=indices,
-        trigger_action_name=trigger_action_message,
-        trigger_action_message=trigger_action_subject
-    )
-
-
 def lambda_handler(event, context, physicalResourceId=None, noEcho=False):
     '''
 

--- a/set_configuration.py
+++ b/set_configuration.py
@@ -327,11 +327,7 @@ def set_monitor(
     trigger_condition_source='ctx.results[0].hits.total.value > 5',
     trigger_action_name=None,
     trigger_action_subject='Monitor Triggered',
-    trigger_action_message='Monitor detected {} satisfying {} within {}'.format(
-        monitor_query_terms,
-        trigger_condition_source,
-        schedule_interval
-    ),
+    trigger_action_message='Monitor detected satisfying condition',
     trigger_action_throttle_enabled='false',
     headers=json.loads(os.getenv('Headers', '{"Content-Type": "application/json"}').strip())
 ):

